Ocean klasse generer

mikk7867 opened the pull request:
please review

mikk7867 committed:
map initial version ting

mikk7867 committed:
udkommentering af ikke-færdige ting så koden kan køres

Ironfirst committed:
initial commit på map repository

Slaite-crate committed:
Hex done

mikk7867 committed:
koordinater klasse

Ironfirst committed:
initial commit på map repository

Ironfirst committed:
initial commit på map repository

Ironfirst committed:
Merge remote-tracking branch 'origin/ocean.hex' into oceanKlasseGenerer

Ironfirst committed:
Merge remote-tracking branch 'origin/ocean.koordinat' into oceanKlasseGenerer

# Conflicts:
#	pom.xml

Ironfirst committed:
initial commit på map repository
hex og cord implementeret

Ironfirst committed:
initial commit på map repository
hex og cord implementeret

Ironfirst committed:
initial commit på map repository
hex og cord implementeret

Ironfirst committed:
initial commit på map repository
hex og cord implementeret

Ironfirst committed:
initial commit på map repository
hex og cord implementeret

Nord og syd er implementere

mikk7867 committed:
finde alle naboer

Ironfirst committed:
size metoden er blevet fjernet og opret ocean metoden er blevet generisk

+ få test er lavet

mikk7867 committed:
namechange + kommentar

Ironfirst committed:
size metoden er blevet fjernet og opret ocean metoden er blevet generisk

+ få test er lavet
test lavet færdig

mikk7867 committed:
skiftet navnet på test klassen

Fuzziwig reviewed (changes requested):
se https://github.com/erik1500/sas1/pull/4

Ironfirst committed:
en del metoder lavet private samt nedarv på test class fjernet

Ironfirst committed:
test classe navngivning ændret,

IllegalArgumentException til oceanX og oceanY er lavet så
værdierne ikke kan være under 1 ved oprettelse

Ironfirst committed:
test classe navngivning ændret,

IllegalArgumentException til oceanX og oceanY er lavet så
værdierne ikke kan være under 1 ved oprettelse

Fuzziwig committed:
Ocean reviewed with few minor changes

Fuzziwig committed:
Ocean reviewed with few minor changes

CeciNB committed:
Merge branch 'master' into oceanKlasseGenerer
